book_source/developers_guide/Adding-an-Input-Converter.Rmd

```diff
@@ -48,14 +49,62 @@ Names should be `met2CF.<sourcename>` and `met2model.<modelname>`.
 
-The key is to process each type of met data (site, reanalysis, forecast, climate scenario, etc) to the exact same standard. This way every operation after that (extract, gap fill, downscale, convert to a model, etc) will always have the exact same inputs. This will make everything else much simpler to code and allow us to avoid a lot of unnecessary data checking, tests, etc being repeated in every downstream function.
\ No newline at end of file
+The key is to process each type of met data (site, reanalysis, forecast, climate scenario, etc) to the exact same standard. This way every operation after that (extract, gap fill, downscale, convert to a model, etc) will always have the exact same inputs. This will make everything else much simpler to code and allow us to avoid a lot of unnecessary data checking, tests, etc being repeated in every downstream function.
+
+### Adding Single-Site Specific Meteorological Data
+
+Perhaps you have meteorological data specific to one site, with a unique format that you would like to add to PEcAn. Your steps would be to:
+  1. write a script or function to convert your files into the netcdf PEcAn standard
+  2. insert that file as an input record for your site following these [instructions](How to Insert new Input Data)
+  
+### Downloading Met data outside of the workflow
+
+Perhaps you would like to obtain data from one of the sources coupled to PEcAn on its own. To do so you can run PEcAn functions on their own. 
+
+Example 1:
+
+Downloading Amerifluxlbl from Niwot Ridge for the year 2004:
+```
+raw.file <-PEcAn.data.atmosphere::download.AmerifluxLBL(sitename = "US-NR1", 
+                                             outfolder = ".", 
+                                             start_date = "2004-01-01", 
+                                             end_date = "2004-12-31")
+```
+
+If you wanted to convert it to PEcAn Standard you would subsequently execute the following:
+```
+bety = list(user='bety', password='bety',host='localhost', dbname='bety', driver='PostgreSQL',write=TRUE)
+con <- PEcAn.DB::db.open(bety)
+bety$con <- con
+
+in.path <- '.'
+in.prefix <- raw.file$dbfile.name
+outfolder <- '.'
+format.id <- 5000000002
+format <- PEcAn.DB::query.format.vars(format.id=format.id,bety = bety)
+lon <- 105.54
+lat <- 40.03
+format$time_zone <- "America/Chicago"
+PEcAn.data.atmosphere::met2CF.csv(in.path = in.path, 
+                                  in.prefix ="US-NR1_CF", 
+                                  outfolder = ".", 
+                                  start_date ="2004-01-01",
+                                  end_date = "2004-01-01",
+                                  format = format) 
+```
+Note: The format.id is specific to the format type of the raw data. You can look up the format id in the Bety database. 
+
+
+## Cohort and Pool Data
```

Comment:
This title is not informative if you don't already know what we're talking about. I'd go with something like Vegetation Data.

And you'll also want a subset on Soil Data

And both these sections, and the met, should cross reference backward and forward to the instructions on how these types of information are actually USED  https://pecanproject.github.io/pecan-documentation/develop/basic-model-configuration.html